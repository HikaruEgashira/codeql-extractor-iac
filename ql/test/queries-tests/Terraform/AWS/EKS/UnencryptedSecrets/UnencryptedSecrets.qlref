security/Terraform/AWS/EKS/UnencryptedSecrets/UnencryptedSecrets.ql