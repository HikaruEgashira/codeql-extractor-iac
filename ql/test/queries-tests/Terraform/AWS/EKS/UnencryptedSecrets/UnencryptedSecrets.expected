| eks.tf:2:1:8:1 | resource aws_eks_cluster cluster | EKS Unencrypted Secrets |
